### PR TITLE
Invariant checks: don't expand big allocatedSectors bitfields

### PR DESCRIPTION
**Pull request by arajasek**
If this bitfield is too big, don't expand it, and look up values in the bitfield directly -- slower, but acceptable.

### CODE REVIEW

File: builtin/v9/miner/invariants.go [hunk 1]
```diff
@@ -76,7 +78,19 @@ func CheckStateInvariants(st *State, store adt.Store, balance abi.TokenAmount) (
 		err = sectorsArr.ForEach(&sector, func(sno int64) error {
 			cpy := sector
 			allSectors[abi.SectorNumber(sno)] = &cpy
-			acc.Require(allocatedSectorsMap == nil || allocatedSectorsMap[uint64(sno)],
```
Comment (arajasek) [inline builtin/v9/miner/invariants.go:79]: I'm not sure why this was ever okay with allocatedSectorsMap being nil...

File: builtin/v9/check.go [hunk 2]
```diff
@@ -274,6 +274,8 @@ func CheckVerifregAgainstDatacap(acc *builtin.MessageAccumulator, verifregSummar
 	for _, allocation := range verifregSummary.Allocations {
 		pendingAllocationsTotal = big.Add(pendingAllocationsTotal, big.NewIntUnsigned(uint64(allocation.Size)))
 	}
+
+	pendingAllocationsTotal = big.Mul(pendingAllocationsTotal, verifreg.DataCapGranularity)
```
Comment (arajasek) [inline builtin/v9/check.go:278]: Drive-by fix.

**Review by geoff-vball — APPROVED** [review]
